src/polyswarm_api/types/schemas/artifact.py

```diff
@@ -0,0 +1,60 @@
+artifact_instance_schema = {
```

Comment:
I think these schema checks should happen on server-side. The client should be flexible and not break with schema validation if the request succeeded.

Reply:
Unless the user requests that it break when the schema is violated, it will not by default.